### PR TITLE
feature/EPL-1303: How to add a field to a Window Tab

### PR DESCRIPTION
**Pull request by Tomas-Vesco**

### CODE REVIEW

File: docs/developer-guide/etendo-classic/how-to-guides/How_to_add_a_field_to_a_Window_Tab.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+and automatically place the field into the module.
+
+![](/assets/developer-guide/etendo-classic/how-to-guides/How_to_add_a_field_to_a_Window_tab_1.png)
+
+As a last step, the Synchronize Terminology process should be run to
+synchronize our newly added fields to the elements created for the columns
+they are based on, so that the UI labels for the new fields get the names
+defined in those elements.
+
+If this window should also be used in _classic UI mode_ then now _ant
```
Comment (isaiasb-etendo) [inline docs/developer-guide/etendo-classic/how-to-guides/How_to_add_a_field_to_a_Window_Tab.md:39]: `./gradlew smartbuild`

File: docs/developer-guide/etendo-classic/how-to-guides/How_to_add_a_field_to_a_Window_Tab.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  - Etendo Classic
+  - Customization
+  - UI Design
+  - Application Dictionary
+  - Modules
+---
+
+# How to add a field to a Window Tab
+
+## Objective
```
Comment (isaiasb-etendo) [inline docs/developer-guide/etendo-classic/how-to-guides/How_to_add_a_field_to_a_Window_Tab.md]: Overview

File: docs/developer-guide/etendo-classic/how-to-guides/How_to_add_a_field_to_a_Window_Tab.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+As a last step, the Synchronize Terminology process should be run to
+synchronize our newly added fields to the elements created for the columns
+they are based on, so that the UI labels for the new fields get the names
+defined in those elements.
+
+If this window should also be used in _classic UI mode_ then now _ant
+smartbuild_ should be execute to compile the changed window.
+
+To see the changes in the new UI just switch role away from the _System
```
Comment (isaiasb-etendo) [inline docs/developer-guide/etendo-classic/how-to-guides/How_to_add_a_field_to_a_Window_Tab.md:42]: To see the changes in the new user interface just change the System Administrator role used for this section (Select a role where you have access to the window) and go to the modified window to see that it has the new field as shown here:

File: docs/developer-guide/etendo-classic/how-to-guides/How_to_add_a_field_to_a_Window_Tab.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+## Objective
+
+The objective of this section is to show how you can add a new field to a tab in Etendo Classic.
+
+It is closely related to the previous [How to add Columns to a Table](/developer-guide/etendo-classic/how-to-guides/How_to_add_Columns_to_a_Table/) section which showed how to add new columns to a table.
+
+### Add the fields to the window
+
+The process to add new fields to an existing window is the same as the one to add fields to a new (empty) window. Going to Application Dictionary | Window, Tabs and Fields, we need to search for the _Production Run_ window. Then, for that window, mark its _Incidence_ tab.
```
Comment (isaiasb-etendo) [inline docs/developer-guide/etendo-classic/how-to-guides/How_to_add_a_field_to_a_Window_Tab.md:21]: `Application Dictionary` > ` Window, Tabs and Fields`

**Comment by Tomas-Vesco** [comment]
@isaiasb-etendo Changes apply.

**Review by RomanMagnoli — APPROVED** [review]

File: docs/developer-guide/etendo-classic/how-to-guides/How_to_add_a_field_to_a_Window_Tab.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+## Overview
+
+The objective of this section is to show how you can add a new field to a tab in Etendo Classic.
+
+It is closely related to the previous [How to add Columns to a Table](/developer-guide/etendo-classic/how-to-guides/How_to_add_Columns_to_a_Table/) section which showed how to add new columns to a table.
+
+The task of adding columns to an existing window is now very simple since most of the work (like defining references, elements) has already been done when adding the columns to the table.
+
+In this HowTo we will show how to place new fields in a window.
```
Comment (isaiasb-etendo) [inline docs/developer-guide/etendo-classic/how-to-guides/How_to_add_a_field_to_a_Window_Tab.md:21]: The commentary explaining what will be done below is missing.

**Review by isaiasb-etendo — APPROVED** [review]

**Pull request merged**